integration-cli/docker_api_images_test.go

```diff
@@ -120,13 +121,16 @@ func (s *DockerSuite) TestAPIImagesHistory(c *check.C) {
 func (s *DockerSuite) TestAPIImagesImportBadSrc(c *check.C) {
 	testRequires(c, Network)
```

Comment:
I think we can remove that one, right ? (as now we don't need to access to network)